Breath_Mask/Breath_Mask.py

```diff
@@ -13,75 +13,88 @@
 # 1 - Temperature - Circuit Playground Built-In LEDs
 # 2 - Total Volatile Organic Compounds [strip]
 # 3 - Co2 Output - NeoPixel [strip]
-num_leds = 8 
+num_leds = 8
 temperature_pix = neopixel.NeoPixel(board.NEOPIXEL, num_leds, brightness=.1)
 tvoc_pix = neopixel.NeoPixel(board.A1, num_leds, bpp=4, brightness=.1)
 co2_pix = neopixel.NeoPixel(board.A2, num_leds, bpp=4, brightness=.1)
-led_draw = .05      # delay for LED pixel turn on/off
+led_draw = .05  # delay for LED pixel turn on/off
 
 # wait for the sensor to be ready and calibrate the thermistor
 while not ccs.data_ready:
     pass
 temp = ccs.temperature
 ccs.temp_offset = temp - 25.0
 
-# clear all LEDs for breathing effect 
+
+# clear all LEDs for breathing effect
+
+
```

Comment:
Can we disable the two lines before methods requirement?

Reply:
I can add that

Reply:
Just checked this. Surprisingly pylint doesn't care how many blank lines there are before a function. The blank lines were caused by me running a PEP8 auto-formatter on the file. I've moved the comments onto the first line of the functions instead. Let me know if that's OK.